fmt/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "forge-fmt"
+version = "0.2.0"
```

Comment:
Is this still accurate?

Reply:
What version should it have if we rename it to `solang-fmt`?

Reply:
If we are going to publish it, intuitively should be the same version as solang-parser and solang?

Reply:
However if we just want to vendor it we should keep the original version I think

Reply:
How about we just vendor it for now. If we want to publish it, we can always rename it and give it a new version.